data/kramdown-rfc2629.erb

```diff
@@ -12,14 +12,25 @@
 <% end -%>
 ]>
 
-<% ps.arr("pi", false) do |pi, val| -%>
-<?rfc <%=pi%>="<%= {true => "yes", false => "no", nil => "yes"}[val] || val %>"?>
+<%
+  rfcattrs = ps.attrs("ipr", "docName=docname", "category=cat",
+                      "consensus", "submissionType=submissiontype", "xml:lang=lang",
+                      "number", "obsoletes", "updates", "seriesNo=seriesno")
+  pis = KramdownRFC::ParameterSet.new({})
+  ps.arr("pi", false) do |pi, val|
+    pis[pi] = {nil => "true", false => "false", "true" => "true"}[val] || val
+  end
+  if piattrs = pis.attrs("tocDepth=tocdepth", "tocInclude=toc",
```

Comment:
This is always true, need a `(...) != ""` (or preferably two separate lines).

Reply:
Oh, and we probably want to build constants like GROUPS out of these mapping hashes instead of building them inline.

Reply:
Adding a default_proc to these hashes would also gets rid of the funky `||`.